code/game/objects/items/misc.dm

```diff
@@ -21,4 +21,11 @@
 	throwforce = 0.0
 	throw_speed = 1
 	throw_range = 20
-	flags = CONDUCT
\ No newline at end of file
+	flags = CONDUCT
+
+/obj/item/weapon/beach_ball/basketball
```

Comment:
It's `/obj/item/weapon/beach_ball/holoball` which..should probably be renamed since it's just a basketball which happens to be only used in the holodeck currently.